### ISSUE TITLE
environment variable `VERGEN_BUILD_DATE` not defined

### ISSUE THREAD

**Issue by williamdes** [rustyhorde/vergen#137]
I switched to `VERGEN_BUILD_TIMESTAMP` 

See: https://docs.rs/vergen/latest/vergen/#environment-variables

```toml
[build-dependencies]
anyhow = "1.0.63"
vergen = { version = "7.4.2", default-features = false, features = ["build", "git"] }
```

Also, could we not need `anyhow` ?

**Comment by crumblingstatue** [comment]
`VERGEN_BUILD_DATE` and `VERGEN_COMMIT_DATE` are missing for me as well, so I'm forced to use timestamps instead.

**Comment by crumblingstatue** [comment]
Temporary workaround I'm eploying: `env!("VERGEN_GIT_COMMIT_TIMESTAMP").split('T').next().unwrap_or("error")`

**Comment by leighmcculloch** [comment]
I'm seeing the same with `VERGEN_COMMIT_DATE`, it is always blank.

**Comment by leighmcculloch** [comment]
I discovered the reason `VERGEN_COMMIT_DATE` was always blank is because I have to explicitly set:
```
*config.git_mut().commit_timestamp_kind_mut() = TimestampKind::DateOnly;
```

**Comment by CraZySacX** [comment]
This should be resolved with the release of [`8.0.0-beta.0`](https://crates.io/crates/vergen/8.0.0-beta.0).  This release should give you more flexibility with the date/timestamps.   Also, I've reduced the number of dependencies I've been using.   Feel free to re-open if you still have issues.
